### ISSUE TITLE
"Remote server" is a mis-nomer in information resources

### ISSUE THREAD

**Issue by finnianr** [cherokee/webserver#1202]
It is possible to setup up a FastCGI connection using a Unix socket by entering a socket address into a "Remote server" information source. But by definition a Unix socket is on local host only, so I submit to you that this title is a misnomer and should be changed to something else.

Also using Unix socket needs to be better documented in the "Cherokee Cookbook". It took me a day and a half to work out how to do Unix sockets. I have posted [a brief guide](https://groups.google.com/forum/#!topic/eiffel-users/uR0PgrqymFA) in the Eiffel language user group.

**Comment by skinkie** [comment]
http://cherokee-project.com/doc/config_info_sources.html See Unix Sockets.

**Comment by finnianr** [comment]
Yes, I did in fact read that section, but what it doesn't make clear is whether it will work "Remote server". It is natural to assume not. And there is no discussion of how to setup relevant permissions. So I regard the documentation as incomplete. I came close to giving up on the idea completely because of this.
